src/crypto/OutgoingRoomKeyRequestManager.js

```diff
@@ -41,6 +41,9 @@ const ROOM_KEY_REQUEST_STATES = {
 
     /** request sent, awaiting reply */
     SENT: 1,
+
+    /** reply received, cancellation not yet sent */
+    CANCELLATION_PENDING: 2,
```

Comment:
The state machine is:

```
-> UNSENT -----> SENT -----> CANCELLATION_PENDING ---->  (deleted)
       |                                                                                                     ^
      +------------------------------------------------------------------------------------+
```

I'll do some ascii art in the file.


> Does it matter that SENT can mean "sent cancellation" AND "sent request"?

it doesn't mean that.